internal/commands/resolve.go

```diff
@@ -114,33 +115,35 @@ func ResolveIncidentByDialog(
 		StatusPageUrl:       statusPageURL,
 	}
 
+	logWriter := app.Logger.With(
+		log.NewValue("incident", incident),
```

Comment:
The ChannelID information is already available here, so we can add it to logWriter fields. Notice that you removed it from the GetIncident error log but did not add it back anywhere.